src/main/java/org/hmhb/program/Program.java

```diff
@@ -0,0 +1,159 @@
+package org.hmhb.program;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.validation.constraints.NotNull;
+
+import java.util.Date;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.hmhb.organization.Organization;
+
+@Entity
+public class Program {
```

Comment:
Some place in the work flow we need to translate an address to lat/long coordinates. If we don't trust the client (a good idea) then the client POSTed Program will not have this Location data, but it would be in the object sent to the client when showing published Programs on a map.

Doesn't have to be part of this pull-request, just starting a dialog.